deps/rabbit/test/rabbit_stream_queue_SUITE.erl

```diff
@@ -1256,7 +1239,7 @@ recover_after_leader_and_coordinator_kill(Config) ->
 
 
     ct:pal("sys state ~p", [CState]),
-
+    rabbit_ct_broker_helpers:rpc(Config, 0, ?MODULE, delete_testcase_queue, [Q]),
```

Comment:
Is this a drive-by change? If this is a fix, should it be backported separately of this whole patch?

Reply:
As I already wrote in Slack:
> Test cases rebalance and restart_stream were always skipped prior to this PR because the feature flag restart_streams in these test cases is misspelled. So, that part should be backported to v3.13.x

This is not a "drive-by change" because this PR removes the code to enable the feature flag in https://github.com/rabbitmq/rabbitmq-server/pull/11659/files#diff-bd1bb71aee8de223a400a433496cefbd338052929a9ed71906665ca953b72746L2820-L2827 which causes test case `rebalance` to run for the first time. The test case was buggy. One bug is the line you are commenting: Test cases didn't clean up properly after each other which caused assertions in test case `rebalance` to fail. (In general, I prefer to randomise test case executions within a group such that test interference can be easily spotted.) So, these test case fixes are already in a separate commit in this PR and can be easily backported to `v3.13.x`.

Reply:
Here is the commit I'm referring to: https://github.com/rabbitmq/rabbitmq-server/pull/11659/commits/6cdfb62a81c287f017a0b8907ee98ab759430583

Reply:
Thank you, I understand now.